### PR TITLE
Consul + Vault - Gossip Encryption

### PR DESCRIPTION
**Pull request by danielehc**
Lab for Gossip Encryption with Consul and Vault.

I'm working on a scenario that automatically rotates the key and I am at a good point, we can merge this if we want and I'll update it once we checked the new version.

### CODE REVIEW

File: consul-secure-gossip-vault/index.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            {"name": "Vault UI", "port": 8200}
+        ],
+        "terminals": [
+            {"name": "server", "target": "host01"}
+        ]
+    },
+    "backend": {
+        "port": 8080,
+        "imageid": "ubuntu"
+    }
```
Comment (yhyakuna) [inline consul-secure-gossip-vault/index.json:68]:
```suggestion
    },
    "private": true
```

**Review by yhyakuna — APPROVED** [review]
LGTM

File: consul-secure-gossip-vault/step1.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Example output:
+
+```
+Success! Enabled the kv secrets engine at: kv-v1/
+```
+
+Once the secret engine is enabled, verify it this using
+the following command:
+
+`vault secrets list -detailed`{{execute T1}}
```
Comment (kaitlincarter-hc) [inline consul-secure-gossip-vault/step1.md:16]: This command is really great for troubleshooting, but in the little katacoda terminal the output is a little difficult to read. Can we just list the engines without all the details?

File: consul-secure-gossip-vault/step1.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### Generate Consul gossip key
+
+Once you have enabled the Vault secrets engine, you can now
+generate the Consul gossip encryption key.
+
+There are multiple ways to generate a valid gossip encryption key.
+
+* **Method 1: Consul binary**
```
Comment (kaitlincarter-hc) [inline consul-secure-gossip-vault/step1.md:25]:
```suggestion
#### Method 1: Consul binary
```

Hopefully this will improve readability.

**Comment by danielehc** [comment]
Merging this PR to test the lab.

**Pull request merged**